src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -296,9 +309,11 @@ public void execute() {
             log("compiled on " + compileTimestamp, Project.MSG_VERBOSE);
 
             // Check for no arguments
-            if (fileName == null && fileSets.isEmpty()) {
+            if (fileName == null
+                && fileSets.isEmpty()
```

Comment:
Add 4 spaces for indentation to separate conditions from the next part of the login (throw new BuildException).
Example:
```java
if (condition1
        && condition2) {
    throw new Exception();
}
```